.github/workflows/deploy-pipeline-aws.yml

```diff
@@ -54,29 +54,29 @@ jobs:
         python-version: [3.8]
     name: Build and deploy the pipeline
     env:
-      AWS_ACCOUNT_ID: ${{ inputs.awsAccountId || secrets.AWS_ACCOUNT_ID }}
```

Comment:
@belspirit we should keep the AWS_ prefixes here. 

Sorry if that wasn't clear. The parameter names changed in the datateer CLI but should not change in the deployment scripts